lib/pages/settings_pages/components/hashtag.dart

```diff
@@ -142,6 +142,10 @@ class _StringMultilineTagsState extends State<StringMultilineTags> {
                     ),
                     onChanged: inputFieldValues.onTagChanged,
                     onSubmitted: (String newTag){
+                      if (newTag.startsWith('#')) {
+                        newTag = newTag.substring(1);
+                      }
```

Comment:
I just realized that if there's a '#' in the middle of the string, it would generate two tags in the DB.

I was thinking of removing any '#' found in the string by always running `newTag = newTag.replaceAll('#', '');`

What do you think?

Reply:
It depends on how we want to handle it. I guess it's not expected that any user would want to add '#' in the middle of a hashtag. Removing the symbols and combining it into a single hashtag seems like a good solution from the developers perspective